packages/discord.js/src/managers/GuildMemberManager.js

```diff
@@ -206,18 +204,70 @@ class GuildMemberManager extends CachedManager {
    */
   fetch(options) {
     if (!options) return this._fetchMany();
-    const user = this.client.users.resolveId(options);
-    if (user) return this._fetchSingle({ user, cache: true });
-    if (options.user) {
-      if (Array.isArray(options.user)) {
-        options.user = options.user.map(u => this.client.users.resolveId(u));
-        return this._fetchMany(options);
-      } else {
-        options.user = this.client.users.resolveId(options.user);
-      }
-      if (!options.limit && !options.withPresences) return this._fetchSingle(options);
+    const { user, limit, withPresences, cache, force } = options;
+    const resolvedUser = this.client.users.resolveId(user ?? options);
+    if (resolvedUser && !limit && !withPresences) return this._fetchSingle({ user: resolvedUser, cache, force });
+    const resolvedUsers = user?.map?.(users => this.client.users.resolveId(users)) ?? resolvedUser ?? undefined;
```

Comment:
Having "users" as param of a `.map` function looks weird, we should find another name. 
I propose "userId", a UserResolvable is not only "userId" but it looks better 🤔 
```suggestion
    const resolvedUsers = user?.map?.(userId => this.client.users.resolveId(userId)) ?? resolvedUser ?? undefined;
```